app/src/main/java/io/manycore/robotcoder/demo/MainActivity.kt

```diff
@@ -0,0 +1,19 @@
+package io.manycore.robotcoder.demo
+
+import android.R
+import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import com.bumptech.glide.Glide
+
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        //Copy paste from Glide repo on https://github.com/bumptech/glide
+        val imageView: ImageView = findViewById<View>(R.id.my_image_view) as ImageView
+        Glide.with(this).load("http://goo.gl/gEgYUd").into(imageView)
+    }
```

Comment:
Glide is initialized from the main thread. If the image to load is big this will create an ANR. You want to move this to a background thread